public/app/features/storage/ExportView.tsx

```diff
@@ -0,0 +1,141 @@
+import { css } from '@emotion/css';
+import React, { useEffect, useState } from 'react';
+
+import { GrafanaTheme2, isLiveChannelMessageEvent, isLiveChannelStatusEvent, LiveChannelScope } from '@grafana/data';
+import { getBackendSrv, getGrafanaLiveSrv } from '@grafana/runtime';
+import { Button, CodeEditor, Modal, useTheme2 } from '@grafana/ui';
+
+import { StorageView } from './types';
+
+interface ExportStatusMessage {
+  running: boolean;
+  target: string;
+  started: number;
+  finished: number;
+  update: number;
+  count: number;
+  current: number;
+  last: string;
+  status: string;
+}
+
+interface Props {
+  onPathChange: (p: string, v?: StorageView) => void;
+}
+
+export const ExportView = ({ onPathChange }: Props) => {
```

Comment:
NOTE: this is very rough, but just moved from the 'admin' section, not new to this PR